schemas/hs00_event_histogram.fbs

```diff
@@ -0,0 +1,17 @@
+
+file_identifier "hs00";
+
+// Represents the n dimensional histogram
+// Subsets of the histogram are also supported
+table EventHistogram {
+    source: string;           // Source name
+    timestamp: ulong;         // Timestamp (in ns, after epoch)
```

Comment:
We deal with the EPICS epoch of 1990 too, so best to be explicit which you mean.

Reply:
True.